arch/arm64/src/common/arm64_addrenv.c

```diff
@@ -179,8 +179,8 @@ static int create_spgtables(arch_addrenv_t *addrenv)
 
   /* Synchronize data and instruction pipelines */
 
-  ARM64_DSB();
-  ARM64_ISB();
+  __MB();
```

Comment:
@lipengfei28 let's change to UP_MB or up_mb to indicate the barrier macro is the standard API between kernel/driver and arch. and replace ALL SP_XXX with the new macros

Reply:
ok,I will change to UP_MB as standard API,